File: sofd-core-opus-integration/src/main/java/dk/sofd/opus/service/model/Post.java
package dk.sofd.opus.service.model;

import lombok.*;

@Getter
@Setter
@Builder
@ToString
@NoArgsConstructor
@AllArgsConstructor
@EqualsAndHashCode(exclude = { "prime" })
public class Post {
	private String street;
	private String localname;
	private String postalCode;
	private String city;
	private String country;
	private boolean addressProtected;
	private boolean prime;
	private String master;
	private String masterId;
}

File: sofd-core-odata/SOFDCore.ODataApi/Models/Post.cs
using System.ComponentModel.DataAnnotations;

namespace SOFDCore.ODataApi.Models
{
    public abstract class Post
    {
        [Key]
        public long Id { get; set; }
        public bool? Prime { get; set; }
        public string Street { get; set; }
        public string Localname { get; set; }
        public string PostalCode { get; set; }
        public string City { get; set; }
        public string Country { get; set; }
        public bool? AddressProtected { get; set; }
        public string Master { get; set; }
        public string MasterId { get; set; }
    }
}
